cmd/sriovdp/manager.go

```diff
@@ -16,59 +16,109 @@ package main
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
+	"io"
 	"os"
+	"path/filepath"
 
 	"github.com/golang/glog"
 	"github.com/jaypipes/ghw"
 
+	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/cdi"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/factory"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/types"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/utils"
 )
 
 const (
-	socketSuffix = "sock"
+	defaultConfig = "/etc/pcidp/config.json"
+	socketSuffix  = "sock"
 )
 
+// cliParams presents CLI parameters for SR-IOV Network Device Plugin
 type cliParams struct {
 	configFile     string
 	resourcePrefix string
+	useCdi         bool
+	hostConfigDir  string
 }
 
-type resourceManager struct {
+// ResourceManager manages resources for SR-IOV Network Device Plugin binaries
+type ResourceManager struct {
 	cliParams
 	pluginWatchMode bool
 	rFactory        types.ResourceFactory
-	configList      []*types.ResourceConfig
+	ConfigList      []*types.ResourceConfig
 	resourceServers []types.ResourceServer
 	deviceProviders map[types.DeviceType]types.DeviceProvider
 }
 
-func newResourceManager(cp *cliParams) *resourceManager {
+// FlagInit parse command line flags
+func FlagInit(cp *cliParams) {
+	flag.StringVar(&cp.configFile, "config-file", defaultConfig,
+		"JSON device pool config file location")
+	flag.StringVar(&cp.resourcePrefix, "resource-prefix", "intel.com",
+		"resource name prefix used for K8s extended re"+
+			"source")
+	flag.BoolVar(&cp.useCdi, "use-cdi", false,
+		"Use Container Device Interface to expose devices in containers")
+	flag.StringVar(&cp.hostConfigDir, "host-config-dir", "/host/etc/pcidp/",
```

Comment:
we dont need this one for now

Reply:
maybe we would want to have : cdiSpecDir, but for now mounting to standard CDI paths is enough (up 2 u if you want to add)

Reply:
> we dont need this one for now

We need the config to understand which devices should be included into the resource pool